Godeps/Godeps.json

```diff
@@ -586,32 +586,48 @@
 			"Rev": "137b4975ecab6e1f0c24c1e3c228a50a3cfba75e"
 		},
 		{
+			"ImportPath": "github.com/containers/image/docker",
+			"Rev": "106607808da3cff168be56821e994611c919d283"
+		},
+		{
+			"ImportPath": "github.com/containers/image/docker/archive",
+			"Rev": "106607808da3cff168be56821e994611c919d283"
+		},
+		{
+			"ImportPath": "github.com/containers/image/docker/daemon",
```

Comment:
The `archive` and `daemon` subpackages should not be necessary. (Perhaps including them here is easier, I don’t know.)

Reply:
(Many more unneeded sub packages, starting with `c/i/transports/alltransports`, have also been added, if that matters to you.)